isb_web/templates/thing.html

```diff
@@ -23,6 +23,61 @@
   </link>
   <script type="module"
     src="https://cdn.jsdelivr.net/gh/isamplesorg/isamples_webui@develop/record-page/src/js/index.js"></script>
+  <script src="https://hypothes.is/embed.js" async></script>
+  <script>
+async function fetchGrantToken(authority, isamples_jwt_url, hypothesis_api_url, login_url, logout_url) {
```

Comment:
I tried to include this in the `index.js` where we source the other JS for this page.  However, I couldn't figure out how to call a function in that file from within the html of this page.  Since the page is dynamically rendered (we need the server URLs to come from the backend that renders the html), we need this line to be able to call this function:

```
window.addEventListener('load', fetchGrantToken('{{ authority|safe }}', '{{ isamples_jwt_url|safe }}', '{{ hypothesis_api_url|safe }}', '{{ login_url|safe }}', '{{ logout_url|safe }}'));
```

I tried using `import` statements and `export` in the hosted JS but was unable to come up with the proper incantation.  Not quite as clean as locating it there, but I didn't want to waste any more time on it.